src/plugins/ncr/utils/hasFieldUpdate.js

```diff
@@ -0,0 +1,45 @@
+import isEqual from 'lodash/isEqual';
+
+const isEmpty = (value) => !value || !value.toString().trim().length;
+
+const findEmptyFieldColumnValue = (value, initialValue, fieldType) => {
+  if (fieldType !== 'FieldArray' || isEmpty(value)) {
+    return null;
+  }
+
+  const fieldColumns = Object.keys(value[0] || {});
+  for (let rowIndex = 0; rowIndex < value.length; rowIndex += 1) {
+    for (let columnIndex = 0; columnIndex < fieldColumns.length; columnIndex += 1) {
+      const column = fieldColumns[columnIndex];
+      const fieldItemValue = value[rowIndex][column];
+      if (initialValue) {
+        const initialValueRow = initialValue[rowIndex] || [];
+        const initialFieldItemValue = initialValueRow[column];
+        if (initialFieldItemValue === '1' && fieldItemValue === '0') {
+          return column;
+        }
+      }
+
+      if (isEmpty(fieldItemValue)) {
+        return column;
+      }
+    }
+  }
+  return null;
+};
+
+export default (formProps, invalidFields = []) => {
+  const initialValues = formProps.initialValues;
+  const values = formProps.values;
+  const registeredFields = formProps.registeredFields || {};
```

Comment:
@hunyoboy this can be:

`const { initialValues, registeredFields = {}, values } = formProps;`

Reply:
@rsumilang definitely thanks. updated.